app.rb

```diff
@@ -1,2 +1,21 @@
 require 'sinatra'
-require 'sinatra/reloader'
\ No newline at end of file
+require 'sinatra/reloader'
+
+# get '/' do
+#   @phrases = ["Hi", "bye"]
+#   erb(:index)
+# end
+
+get '/' do
+
+  return "99 bottles of beer on the wall <a href='98'> Take one down</a>"
+end
+
+get "/:num_bottles" do
+  num2 = params[:num_bottles]
+  if num2 == "0"
+    return "NO MORE BOTTLES! <a href='/'> Start over?</a>"
+  end
+  #  num2 + " bottles of beer on the wall <a href="+(num2.to_i-1).to_s+">Take one down</a>"
+  "#{num2} bottles of beer on the wall <a href='/#{(num2.to_i - 1).to_s}'> Take one down</a>"
+end
```

Comment:
I encourage you commit more frequently, so as you improve your solution to delete old code. Lines 4-8 could be deleted.